### ISSUE TITLE
Usage of LHMRA

### ISSUE THREAD

**Issue by vateye** [OpenGVLab/UniFormerV2#1]
I have noticed that 'NO_LMHRA' is enabled in most of the experiments such as K400/K600/K700/K710. Why we should not use LMHRA in the model since it is intuitive that the local temporal cue should be explored during the training.

**Comment by vateye** [comment]
Thanks for your reply. Besides, I have some questions about the initialization and the training. Since you introduce new parameters to the original ViT, I did not see anywhere you initialize the new parameters with Xavier or Normal (not those initialized with zeros). Meanwhile, the new parameters seem to be share the same learning rate with the ViT, I am wondering why not use larger learning rate for those parameters.

**Comment by Andy1621** [comment]
For most of the new parameters, I just use the default initialization. In fact, the default initialization in PyTorch is enough. You can read the source code for those layers in PyTorch.

For some special layers, I initialize them with zeros, including the last point-wise convolutions in the local temporal MHRA, the query tokens and output projection layers in the query-based cross MHRA, the last linear layers in the FFN of the global UniBlock, and the learnable fusion weights.

https://github.com/OpenGVLab/UniFormerV2/blob/6a678d0b82435fd7f86757cedc35de00bb28ced3/slowfast/models/uniformerv2_model.py#L53-L56

https://github.com/OpenGVLab/UniFormerV2/blob/6a678d0b82435fd7f86757cedc35de00bb28ced3/slowfast/models/uniformerv2_model.py#L153-L159

https://github.com/OpenGVLab/UniFormerV2/blob/6a678d0b82435fd7f86757cedc35de00bb28ced3/slowfast/models/uniformerv2_model.py#L219

Such zero initialization can main the original input, thus making the training stable at first. More importantly, we use a relatively small learning rate (e.g., 2e-5) compared to the previous work. Thus we do not need to decrease/increase the learning rate for those new parameters. BTW, in our experiments, changing the learning rate scale does not bring any improvement.

**Comment by vateye** [comment]
Thanks!
